runners/wikiConsole/templates/createFile.asciidoc

```diff
@@ -0,0 +1,28 @@
+== Create the file <%= fileName; %>
+
+==== Prerequisites
+* Existing folder you want to load the file. (If the folder doesn't exist you can create it from terminal.)
+
+
+.Operating system
+. Windows + 
+Execute the command to create a folder if not already existing `New-Item -ItemType Directory -Force -Path <%= filePath; %>`.
+. Linux and macOS + 
+Execute the command `mkdir -p <%= filePath; %>`. The `-p` flag won't throw an error if the target folder already exists and allows the creation of parent directories.
+
+=== Create <%= fileName; %> from command prompt
+First, you need to open a command prompt in your current workspace. For Windows, you can use PowerShell and on Linux, you can use the Terminal.
+
+.Operating System
+. Windows + 
+In Powershell, we use the command *New-Item* to create a new file<% if(contentPath) { %>, and to insert some content into the created file we use the *Get-Content* command<% } %>.
+More information about the *New-Item* function and the usage can be found https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/new-item?view=powershell-7.1[here]<% if(contentPath) { %> and some information about the *Get-Content* function and the usage can be found https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/set-content?view=powershell-7.1[here]<% } %>. 
+To create a new empty <%= fileName; %> <% if(contentPath){ %>and insert some data from into it, <%= contentFile; %> <% } %>you can execute the command `New-Item  -Path "<%= filePath; %>" -Name 
```

Comment:
Is here a word missing? `To create a new empty <%= fileName; %>`

Reply:
_To create a new empty CustomerEntity.java and write some data from CustomerEntity.java into it, you can execute the command `cat <PATH>`_ Would be the created sentence and if you don't pass a content file its just: _To create a new empty CustomerEntity.java you can execute the command `cat <PATH>`_ . Do you want to have file before or after the filename ?

Reply:
See my other comment https://github.com/devonfw-tutorials/tutorial-compiler/pull/215#discussion_r635212715